pulp_python/app/models.py

```diff
@@ -23,51 +23,42 @@ class Classifier(models.Model):
     """
 
     name = models.TextField()
-    pythonpackagecontent = models.ForeignKey("PythonPackageContent", related_name="classifiers",
+    python_package_content = models.ForeignKey("PythonPackageContent", related_name="classifiers",
                                              related_query_name="classifier")
 
 
 class PythonPackageContent(Content):
     """
     A Content Type representing Python's Distribution Package as
-    defined in pep-0426
+    defined in pep-0426 and pep-0345
+    https://www.python.org/dev/peps/pep-0491/
+    https://www.python.org/dev/peps/pep-0345/
     """
 
     TYPE = 'python'
     filename = models.TextField(unique=True, db_index=True, blank=False)
-    path = models.TextField(blank=False)
```

Comment:
Removed path, this is something not part of the metadata. The information is the same as Pulp's artifact relative_path